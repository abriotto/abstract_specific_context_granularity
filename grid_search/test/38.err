  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s] 11%|####################7                                                                                                                                                                      | 11/99 [00:00<00:00, 96.38it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:00<00:00, 81.29it/s] 30%|########################################################6                                                                                                                                  | 30/99 [00:00<00:01, 65.84it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:00<00:00, 67.97it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:00<00:00, 69.19it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:00<00:00, 64.53it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:00<00:00, 64.26it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:01<00:00, 63.91it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:01<00:00, 68.80it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:01<00:00, 67.37it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:01<00:00, 59.09it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 66.90it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 24%|#############################################1                                                                                                                                              | 6/25 [00:00<00:00, 54.50it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:00<00:00, 64.48it/s] 92%|############################################################################################################################################################################               | 23/25 [00:00<00:00, 70.79it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 66.25it/s]
