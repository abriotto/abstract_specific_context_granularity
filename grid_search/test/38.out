# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=2"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.69306880235672, "acc": 0.5276556611061096, "length": 3.605739116668701, "mode": "train", "epoch": 1}
{"loss": 0.6897331476211548, "acc": 0.5504908561706543, "length": 1.082417607307434, "mode": "test", "epoch": 1}
{"loss": 0.6899732947349548, "acc": 0.5495570302009583, "length": 3.543912649154663, "mode": "train", "epoch": 2}
{"loss": 0.6886898875236511, "acc": 0.5576779246330261, "length": 1.005494475364685, "mode": "test", "epoch": 2}
{"loss": 0.6879451274871826, "acc": 0.5419267416000366, "length": 3.3823680877685547, "mode": "train", "epoch": 3}
{"loss": 0.6862542033195496, "acc": 0.5589117407798767, "length": 1.2967033386230469, "mode": "test", "epoch": 3}
{"loss": 0.6824819445610046, "acc": 0.5640507340431213, "length": 3.1398961544036865, "mode": "train", "epoch": 4}
{"loss": 0.6806960105895996, "acc": 0.5862952470779419, "length": 1.6263736486434937, "mode": "test", "epoch": 4}
{"loss": 0.6648602485656738, "acc": 0.615602970123291, "length": 2.840219497680664, "mode": "train", "epoch": 5}
{"loss": 0.6571258902549744, "acc": 0.6329874396324158, "length": 2.1318681240081787, "mode": "test", "epoch": 5}
{"loss": 0.6263018250465393, "acc": 0.6863762736320496, "length": 2.8403048515319824, "mode": "train", "epoch": 6}
{"loss": 0.6282232999801636, "acc": 0.6591482758522034, "length": 2.219780206680298, "mode": "test", "epoch": 6}
{"loss": 0.5902167558670044, "acc": 0.701779305934906, "length": 3.005875825881958, "mode": "train", "epoch": 7}
{"loss": 0.6039484143257141, "acc": 0.6842535138130188, "length": 3.0, "mode": "test", "epoch": 7}
{"loss": 0.5554226040840149, "acc": 0.7151241302490234, "length": 3.3827311992645264, "mode": "train", "epoch": 8}
{"loss": 0.5852221250534058, "acc": 0.6928132176399231, "length": 3.4065933227539062, "mode": "test", "epoch": 8}
{"loss": 0.5267244577407837, "acc": 0.7421696782112122, "length": 3.4582407474517822, "mode": "train", "epoch": 9}
{"loss": 0.5845690965652466, "acc": 0.6914190053939819, "length": 3.5604395866394043, "mode": "test", "epoch": 9}
{"loss": 0.5140160322189331, "acc": 0.7468194365501404, "length": 3.6169891357421875, "mode": "train", "epoch": 10}
{"loss": 0.5693597197532654, "acc": 0.7190902829170227, "length": 3.8021976947784424, "mode": "test", "epoch": 10}
{"loss": 0.5010893940925598, "acc": 0.7585200071334839, "length": 3.68752384185791, "mode": "train", "epoch": 11}
{"loss": 0.5560049414634705, "acc": 0.702380895614624, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.4884244501590729, "acc": 0.7735092043876648, "length": 3.885042428970337, "mode": "train", "epoch": 12}
{"loss": 0.5555214285850525, "acc": 0.7316849231719971, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.4771607220172882, "acc": 0.772287130355835, "length": 3.9270548820495605, "mode": "train", "epoch": 13}
{"loss": 0.5359518527984619, "acc": 0.7115384340286255, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4675965905189514, "acc": 0.7753400206565857, "length": 3.959066867828369, "mode": "train", "epoch": 14}
{"loss": 0.5330603718757629, "acc": 0.7335166335105896, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.45801565051078796, "acc": 0.7846269011497498, "length": 3.966721534729004, "mode": "train", "epoch": 15}
{"loss": 0.5371043682098389, "acc": 0.7536631226539612, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.44841399788856506, "acc": 0.7906481027603149, "length": 3.9649698734283447, "mode": "train", "epoch": 16}
{"loss": 0.5196529626846313, "acc": 0.7591575980186462, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.43571367859840393, "acc": 0.8135607838630676, "length": 3.9784576892852783, "mode": "train", "epoch": 17}
{"loss": 0.5268802642822266, "acc": 0.7554945349693298, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.4333356022834778, "acc": 0.8171103000640869, "length": 3.976600170135498, "mode": "train", "epoch": 18}
{"loss": 0.5068943500518799, "acc": 0.7884616851806641, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.41427719593048096, "acc": 0.824621319770813, "length": 3.9852771759033203, "mode": "train", "epoch": 19}
{"loss": 0.5159421563148499, "acc": 0.7673993110656738, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.3982648551464081, "acc": 0.8303345441818237, "length": 3.98345685005188, "mode": "train", "epoch": 20}
{"loss": 0.49766629934310913, "acc": 0.783882737159729, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3868996202945709, "acc": 0.8362867832183838, "length": 3.9913885593414307, "mode": "train", "epoch": 21}
{"loss": 0.49163907766342163, "acc": 0.7967033982276917, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.37597936391830444, "acc": 0.8473150730133057, "length": 3.994018077850342, "mode": "train", "epoch": 22}
{"loss": 0.5017211437225342, "acc": 0.795787513256073, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3714066743850708, "acc": 0.8503637909889221, "length": 3.9928534030914307, "mode": "train", "epoch": 23}
{"loss": 0.4925891160964966, "acc": 0.7829670906066895, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.3610480725765228, "acc": 0.8473084568977356, "length": 3.9926116466522217, "mode": "train", "epoch": 24}
{"loss": 0.4782446026802063, "acc": 0.8104396462440491, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.35902029275894165, "acc": 0.8544283509254456, "length": 3.9973952770233154, "mode": "train", "epoch": 25}
{"loss": 0.47440189123153687, "acc": 0.8159341216087341, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.3584372103214264, "acc": 0.8590026497840881, "length": 3.996771812438965, "mode": "train", "epoch": 26}
{"loss": 0.46840494871139526, "acc": 0.815018355846405, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.3545691668987274, "acc": 0.8611636161804199, "length": 3.9972777366638184, "mode": "train", "epoch": 27}
{"loss": 0.4630067050457001, "acc": 0.8141025900840759, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.3465751111507416, "acc": 0.8587245345115662, "length": 3.99630069732666, "mode": "train", "epoch": 28}
{"loss": 0.4515378177165985, "acc": 0.8223445415496826, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.33620432019233704, "acc": 0.8607650995254517, "length": 3.9937336444854736, "mode": "train", "epoch": 29}
{"loss": 0.47474926710128784, "acc": 0.8214287161827087, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.32957643270492554, "acc": 0.8636333346366882, "length": 3.9980621337890625, "mode": "train", "epoch": 30}
{"loss": 0.4483853280544281, "acc": 0.8269232511520386, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.3339250087738037, "acc": 0.8605826497077942, "length": 3.9990127086639404, "mode": "train", "epoch": 31}
{"loss": 0.46876630187034607, "acc": 0.8168498277664185, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.31809890270233154, "acc": 0.8746752142906189, "length": 3.9991648197174072, "mode": "train", "epoch": 32}
{"loss": 0.46067163348197937, "acc": 0.8141026496887207, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.31591078639030457, "acc": 0.8685328960418701, "length": 3.998187780380249, "mode": "train", "epoch": 33}
{"loss": 0.46390923857688904, "acc": 0.8168498277664185, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.31111010909080505, "acc": 0.876211404800415, "length": 3.9994161128997803, "mode": "train", "epoch": 34}
{"loss": 0.4501161575317383, "acc": 0.824176013469696, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.31083494424819946, "acc": 0.8703901171684265, "length": 3.9991893768310547, "mode": "train", "epoch": 35}
{"loss": 0.46312397718429565, "acc": 0.8250916004180908, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.31372541189193726, "acc": 0.8694753646850586, "length": 3.999507188796997, "mode": "train", "epoch": 36}
{"loss": 0.44931456446647644, "acc": 0.8260074853897095, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.3029716908931732, "acc": 0.8703383207321167, "length": 3.9962925910949707, "mode": "train", "epoch": 37}
{"loss": 0.45665740966796875, "acc": 0.825091540813446, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.29996123909950256, "acc": 0.8771368265151978, "length": 3.9994754791259766, "mode": "train", "epoch": 38}
{"loss": 0.4667612910270691, "acc": 0.8012821674346924, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2964351773262024, "acc": 0.8811160326004028, "length": 3.99920654296875, "mode": "train", "epoch": 39}
{"loss": 0.45220959186553955, "acc": 0.826923131942749, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.29611483216285706, "acc": 0.8783392906188965, "length": 3.9985337257385254, "mode": "train", "epoch": 40}
{"loss": 0.44171959161758423, "acc": 0.8260073661804199, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.2854310870170593, "acc": 0.8859010338783264, "length": 3.9962823390960693, "mode": "train", "epoch": 41}
{"loss": 0.4399539828300476, "acc": 0.8260071873664856, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.28375473618507385, "acc": 0.8838810920715332, "length": 3.9998106956481934, "mode": "train", "epoch": 42}
{"loss": 0.41726696491241455, "acc": 0.826923131942749, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.27517643570899963, "acc": 0.8893933296203613, "length": 3.9997406005859375, "mode": "train", "epoch": 43}
{"loss": 0.4071506857872009, "acc": 0.8379120230674744, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.2699004113674164, "acc": 0.8921214938163757, "length": 3.998990774154663, "mode": "train", "epoch": 44}
{"loss": 0.44737881422042847, "acc": 0.8205129504203796, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.2622884511947632, "acc": 0.8919634819030762, "length": 3.993001937866211, "mode": "train", "epoch": 45}
{"loss": 0.44080954790115356, "acc": 0.8205127716064453, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.2602744698524475, "acc": 0.8930662274360657, "length": 3.999593496322632, "mode": "train", "epoch": 46}
{"loss": 0.4463720917701721, "acc": 0.8122710585594177, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.25410452485084534, "acc": 0.894605278968811, "length": 3.9999094009399414, "mode": "train", "epoch": 47}
{"loss": 0.4412970244884491, "acc": 0.8223443627357483, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.26017558574676514, "acc": 0.897047221660614, "length": 3.999732494354248, "mode": "train", "epoch": 48}
{"loss": 0.45227760076522827, "acc": 0.821428656578064, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.25233665108680725, "acc": 0.8964455723762512, "length": 3.999979257583618, "mode": "train", "epoch": 49}
{"loss": 0.44839638471603394, "acc": 0.8232601284980774, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.24962550401687622, "acc": 0.8982701301574707, "length": 3.999581813812256, "mode": "train", "epoch": 50}
{"loss": 0.43836459517478943, "acc": 0.8260073065757751, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.2519179880619049, "acc": 0.8967499136924744, "length": 3.999871253967285, "mode": "train", "epoch": 51}
{"loss": 0.44854605197906494, "acc": 0.817765474319458, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.25269177556037903, "acc": 0.8961313962936401, "length": 3.999379873275757, "mode": "train", "epoch": 52}
{"loss": 0.43099626898765564, "acc": 0.8379121422767639, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.2583848536014557, "acc": 0.8991877436637878, "length": 3.9981937408447266, "mode": "train", "epoch": 53}
{"loss": 0.4138568341732025, "acc": 0.8470696210861206, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.2442876249551773, "acc": 0.904105007648468, "length": 3.999987840652466, "mode": "train", "epoch": 54}
{"loss": 0.43117159605026245, "acc": 0.8250916600227356, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.2582624852657318, "acc": 0.8918135762214661, "length": 3.998002529144287, "mode": "train", "epoch": 55}
{"loss": 0.3889088034629822, "acc": 0.8562271595001221, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.24003590643405914, "acc": 0.9031663537025452, "length": 3.998811721801758, "mode": "train", "epoch": 56}
{"loss": 0.410467267036438, "acc": 0.845238208770752, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.2410348802804947, "acc": 0.9031624794006348, "length": 3.999032497406006, "mode": "train", "epoch": 57}
{"loss": 0.40003079175949097, "acc": 0.8379120826721191, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.2275777906179428, "acc": 0.9046706557273865, "length": 3.998206615447998, "mode": "train", "epoch": 58}
{"loss": 0.3825617730617523, "acc": 0.8507325649261475, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.23003332316875458, "acc": 0.9077674746513367, "length": 3.999396324157715, "mode": "train", "epoch": 59}
{"loss": 0.4221675395965576, "acc": 0.8305861353874207, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.2314099818468094, "acc": 0.9111478924751282, "length": 3.9998998641967773, "mode": "train", "epoch": 60}
{"loss": 0.39273715019226074, "acc": 0.8489012122154236, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.21659675240516663, "acc": 0.9145192503929138, "length": 3.998084545135498, "mode": "train", "epoch": 61}
{"loss": 0.3865538537502289, "acc": 0.8507325649261475, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.2174341082572937, "acc": 0.9123767018318176, "length": 3.999979019165039, "mode": "train", "epoch": 62}
{"loss": 0.42447197437286377, "acc": 0.8470696806907654, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.2220066338777542, "acc": 0.9083937406539917, "length": 3.999974012374878, "mode": "train", "epoch": 63}
{"loss": 0.41775819659233093, "acc": 0.8351649045944214, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.23163890838623047, "acc": 0.9070900678634644, "length": 3.9982640743255615, "mode": "train", "epoch": 64}
{"loss": 0.43500590324401855, "acc": 0.8315019011497498, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.23690520226955414, "acc": 0.9004244804382324, "length": 3.999795436859131, "mode": "train", "epoch": 65}
{"loss": 0.4193733334541321, "acc": 0.8351649045944214, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.22601328790187836, "acc": 0.9132958650588989, "length": 3.9999735355377197, "mode": "train", "epoch": 66}
{"loss": 0.41703546047210693, "acc": 0.8388279676437378, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.21574759483337402, "acc": 0.912985622882843, "length": 3.9998035430908203, "mode": "train", "epoch": 67}
{"loss": 0.3901115357875824, "acc": 0.8543956279754639, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.20680256187915802, "acc": 0.9147830009460449, "length": 3.998319387435913, "mode": "train", "epoch": 68}
{"loss": 0.3887871205806732, "acc": 0.8571429252624512, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.19942998886108398, "acc": 0.9212619662284851, "length": 3.9999866485595703, "mode": "train", "epoch": 69}
{"loss": 0.4084806442260742, "acc": 0.8479853868484497, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.19838280975818634, "acc": 0.9221793413162231, "length": 3.999953269958496, "mode": "train", "epoch": 70}
{"loss": 0.4029022455215454, "acc": 0.8507325649261475, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.19804894924163818, "acc": 0.9224836826324463, "length": 3.999892234802246, "mode": "train", "epoch": 71}
{"loss": 0.3735980987548828, "acc": 0.8589745163917542, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.19212833046913147, "acc": 0.9185042381286621, "length": 3.999979019165039, "mode": "train", "epoch": 72}
{"loss": 0.3934650123119354, "acc": 0.8516483306884766, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.20244571566581726, "acc": 0.918196976184845, "length": 3.999943494796753, "mode": "train", "epoch": 73}
{"loss": 0.38510414958000183, "acc": 0.85347980260849, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.1939043402671814, "acc": 0.9273867011070251, "length": 3.9999353885650635, "mode": "train", "epoch": 74}
{"loss": 0.3941469192504883, "acc": 0.8424909114837646, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.19010098278522491, "acc": 0.9267711639404297, "length": 3.999838352203369, "mode": "train", "epoch": 75}
{"loss": 0.3826754689216614, "acc": 0.855311393737793, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.1825021207332611, "acc": 0.9276959896087646, "length": 3.9999938011169434, "mode": "train", "epoch": 76}
{"loss": 0.42035287618637085, "acc": 0.8424909114837646, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.18458493053913116, "acc": 0.9295322299003601, "length": 3.999945640563965, "mode": "train", "epoch": 77}
{"loss": 0.4004410207271576, "acc": 0.8333332538604736, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.18668442964553833, "acc": 0.924630343914032, "length": 3.9999358654022217, "mode": "train", "epoch": 78}
{"loss": 0.37485742568969727, "acc": 0.8562271595001221, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.1811731457710266, "acc": 0.9298399686813354, "length": 3.9999797344207764, "mode": "train", "epoch": 79}
{"loss": 0.37807735800743103, "acc": 0.8461540341377258, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.17695830762386322, "acc": 0.9313716292381287, "length": 3.9999725818634033, "mode": "train", "epoch": 80}
{"loss": 0.4006582498550415, "acc": 0.8443223834037781, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.174923375248909, "acc": 0.9319591522216797, "length": 3.999521493911743, "mode": "train", "epoch": 81}
{"loss": 0.392258882522583, "acc": 0.8516484498977661, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.16883064806461334, "acc": 0.9344355463981628, "length": 3.9999802112579346, "mode": "train", "epoch": 82}
{"loss": 0.3701702952384949, "acc": 0.8534799814224243, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.1759672313928604, "acc": 0.936199963092804, "length": 3.996365785598755, "mode": "train", "epoch": 83}
{"loss": 0.3757658898830414, "acc": 0.8626373410224915, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.16153495013713837, "acc": 0.9399428963661194, "length": 3.9997503757476807, "mode": "train", "epoch": 84}
{"loss": 0.3847818076610565, "acc": 0.8470695614814758, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.17158126831054688, "acc": 0.9368584752082825, "length": 3.999405860900879, "mode": "train", "epoch": 85}
{"loss": 0.3964781165122986, "acc": 0.8525642156600952, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.1697651445865631, "acc": 0.9401865005493164, "length": 3.9982829093933105, "mode": "train", "epoch": 86}
{"loss": 0.3700883388519287, "acc": 0.848901093006134, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.15959231555461884, "acc": 0.9408698678016663, "length": 3.9999866485595703, "mode": "train", "epoch": 87}
{"loss": 0.40458178520202637, "acc": 0.8516485095024109, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.17658960819244385, "acc": 0.9338225722312927, "length": 3.999974012374878, "mode": "train", "epoch": 88}
{"loss": 0.40823888778686523, "acc": 0.8470696806907654, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.16285406053066254, "acc": 0.9445392489433289, "length": 3.999851942062378, "mode": "train", "epoch": 89}
{"loss": 0.4234340786933899, "acc": 0.855311393737793, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.18206194043159485, "acc": 0.9313721656799316, "length": 3.999987840652466, "mode": "train", "epoch": 90}
{"loss": 0.42918962240219116, "acc": 0.8388278484344482, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.17486807703971863, "acc": 0.9328797459602356, "length": 3.999324083328247, "mode": "train", "epoch": 91}
{"loss": 0.3758965730667114, "acc": 0.8562271595001221, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.1699354499578476, "acc": 0.9319851994514465, "length": 3.999990940093994, "mode": "train", "epoch": 92}
{"loss": 0.42197632789611816, "acc": 0.8461538553237915, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.18199309706687927, "acc": 0.9286147356033325, "length": 3.9999914169311523, "mode": "train", "epoch": 93}
{"loss": 0.4028201997280121, "acc": 0.8507325649261475, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.16953426599502563, "acc": 0.9359643459320068, "length": 3.999877452850342, "mode": "train", "epoch": 94}
{"loss": 0.39507079124450684, "acc": 0.8571427464485168, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.16242749989032745, "acc": 0.9338234663009644, "length": 3.99999737739563, "mode": "train", "epoch": 95}
{"loss": 0.3946155905723572, "acc": 0.8562271595001221, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.15430225431919098, "acc": 0.9408419728279114, "length": 3.999296188354492, "mode": "train", "epoch": 96}
{"loss": 0.3790808618068695, "acc": 0.8543956279754639, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.1546592265367508, "acc": 0.9414820671081543, "length": 3.9999780654907227, "mode": "train", "epoch": 97}
{"loss": 0.3630174994468689, "acc": 0.8626375198364258, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.14992384612560272, "acc": 0.9405618906021118, "length": 3.999951124191284, "mode": "train", "epoch": 98}
{"loss": 0.38825100660324097, "acc": 0.8534799218177795, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.15150263905525208, "acc": 0.9453102350234985, "length": 3.997511863708496, "mode": "train", "epoch": 99}
{"loss": 0.37139269709587097, "acc": 0.860805869102478, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.14951758086681366, "acc": 0.9427080750465393, "length": 3.999990224838257, "mode": "train", "epoch": 100}
{"loss": 0.3867280185222626, "acc": 0.8608059883117676, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.8151341676712036
